src/test/java/domain/LottoTest.java

```diff
@@ -0,0 +1,75 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class LottoTest {
+    @DisplayName("중복이 없는 6개의 숫자로 로또 객체를 생성한다")
+    @Test
+    void createLottoWithDistinctNumbers() {
+        List<Integer> distinctNumbers = Arrays.asList(1, 2, 3, 4, 5, 45);
+
+        assertThatCode(() -> Lotto.of(distinctNumbers))
+                .doesNotThrowAnyException();
+    }
+
+    @DisplayName("6개의 숫자에 중복이 있을 경우 예외를 던진다")
+    @Test
+    void throwExceptionWhenCreateLottoWithDuplicateNumbers() {
+        List<Integer> duplicateNumber = Arrays.asList(1, 2, 3, 4, 6, 6);
+
+        assertThatThrownBy(() -> Lotto.of(duplicateNumber))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("로또 번호는 중복될 수 없습니다.");
+    }
+
+    @DisplayName("NULL일 경우 예외를 던진다")
+    @Test
+    void throwExceptionWithNull() {
+        assertThatThrownBy(() -> Lotto.of(null))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("로또 번호는 6개여야 합니다.");
+    }
+
+    @DisplayName("6개의 숫자에 중복이 있을 경우 예외를 던진다")
+    @Test
+    void throwExceptionIfSizeOfLottoIsNot6() {
+        List<Integer> numbers = Arrays.asList(1, 2, 3);
+
+        assertThatThrownBy(() -> Lotto.of(numbers))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("로또 번호는 6개여야 합니다.");
```

Comment:
여기도 동일하지만 상수 값을 사용하면 값이 변경됐을 때 변경을 덜 해도 되겠죠? ㅎㅎ

```suggestion
                .hasMessage(String.format("로또 번호는 %d개여야 합니다.", LottoProperties.COUNT_OF_NUMBERS));
```